app/src/components/Item/AddItemForm.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+
+const AddItemForm = ({ currentContainerId, handleFormSubmit }) => {
+
+  const data = {
+    item_name: "",
+    additional_information: "",
+    is_container: false
+  };
+
+  const handleNameChange = (event) => {
+    data.item_name = event.target.value;
+  }
+
+  const handleAIChange = (event) => {
```

Comment:
We should avoid to use abbreviation to name a method to achieve better readbility.